src/views/admin/PersonCreationView.vue

```diff
@@ -441,6 +605,72 @@
             ></v-text-field>
           </FormRow>
         </div>
+        <!-- Befristung -->
+        <div
+          class="mt-4"
+          v-if="selectedOrganisation && selectedRolle"
+        >
+          <v-row>
+            <h3 class="headline-3">3. {{ $t('admin.befristung.assignBefristung') }}</h3>
+          </v-row>
+          <FormRow
+            :errorLabel="selectedBefristungProps?.error || ''"
+            labelForId="befristung-select"
+            :isRequired="true"
+            :label="$t('admin.befristung.befristung')"
+          >
+            <v-text-field
+              v-model="selectedBefristung"
+              v-bind="selectedBefristungProps"
+              prepend-icon=""
+              append-inner-icon="$calendar"
+              variant="outlined"
+              placeholder="TT.MM.JJJJ"
+              color="primary"
+            ></v-text-field>
+          </FormRow>
+          <!-- Radio buttons for Befristung options -->
+          <v-row class="align-center">
+            <v-col
+              class="py-0 mt-n1"
+              cols="12"
+              sm="7"
+              offset-sm="5"
+            >
+              <v-radio-group
+                v-model="selectedBefristungOption"
+                v-bind="selectedBefristungOptionProps"
+                @update:modelValue="handleBefristungOptionChange"
+              >
+                <v-radio
+                  :label="`${t('admin.befristung.untilEndOfSchoolYear')} ${getNextSchuljahresende()}`"
```

Comment:
```suggestion
                  :label="`${t('admin.befristung.untilEndOfSchoolYear')} (${getNextSchuljahresende()})`"
```
In Klammern wirkt es schöner